subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,190 @@
+The dependency tree in a link:https://scans.gradle.com/[build scan] shows information about conflicts.
+Click on a dependency and select the "Required By" tab to see the selection reason and origin of the dependency.
 
-.Dependency insight capabilities in a build scan
 image::dependency-management-dependency-insight-report-build-scan.png[]
 
-Every Gradle project provides the task `dependencyInsight` to render the so-called  _dependency insight report_ from the command line.
-Given a dependency in the dependency graph you can identify the selection reason and track down the origin of the dependency selection.
-You can think of the dependency insight report as the inverse representation of the dependency report for a given dependency.
+=== Dependency Insights
 
-The task takes the following parameters:
+Gradle provides the built-in `dependencyInsight` task to render a  _dependency insight report_ from the command line.
+Dependency insights only provide information about a single project at a time.
+Given a dependency, you can identify the selection reason and origin.
+
+`dependencyInsight` accepts the following parameters:
 
 `--dependency <dependency>` (mandatory)::
-Indicates which dependency to focus on.
-It can be a complete `group:name`, or part of it.
-If multiple dependencies match, they are all printed in the report.
+The dependency to generate an insight report for.
+You can supply a complete `group:name`, or part of it.
+If multiple dependencies match, Gradle generates a report covering all matching dependencies.
 `--configuration <name>` (mandatory)::
-Indicates which configuration to resolve for showing the dependency information (_camelCase_ also supported like in `dependencies` task).
-Note that the <<java_plugin#java_plugin, Java plugin>> wires a convention with the value `compileClasspath`, making the parameter optional.
+The dependency configuration from which to supply the dependency information.
+This parameter is optional for projects that use the <<java_plugin#java_plugin, Java plugin>>, since the plugin provides a default value of `compileClasspath`.
 `--single-path` (optional)::
-Indicates to render only a single path to the dependency.
-This might be useful to trim down the output in large graphs.
-
-[NOTE]
-====
-The `dependencyInsight` task selector will only execute the `dependencyInsight` task on a _single_ project.
-If you run the task on the root project, it will show the dependency information of the root project and not of any subproject.
-Be sure to always target the <<command_line_interface#executing_tasks_in_multi_project_builds, right project>> when running `dependencyInsight`.
-====
-
-=== Example: Using the dependency insight report for a given dependency
+Render only a single path to the dependency.
 
-.Output of **`gradle -q dependencyInsight --dependency commons-codec --configuration scm`**
 ----
```

Comment:
This example, without an Example header, feels out of place.

Why not move it after the "Selection reasons" section, since it illustrates both elements: the command and the output.